### PR TITLE
Pull Request

### PR DESCRIPTION
**Pull request by kmtos**
I'm really confused. It says 1000+ added files, but I can't find what files were added. The main thing that was added here was the initial conditions to the Double Sided Gaussian.

### CODE REVIEW

File: Limits/python/Models.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def fit2D(self,ws,hist,name,save=False,doErrors=False,saveDir='', xFitRange=[0,3
         #ws.var('x').setRange('xRange', xFitRange[0], xFitRange[1])
         #ws.var('y').setRange('yRange', yFitRange[0], yFitRange[1])
         #print ("X_FIT_RANGE=", xFitRange, "\tY_FIT_RANGE=", yFitRange)
-        fr = model.fitTo(hist,ROOT.RooFit.Save(),ROOT.RooFit.SumW2Error(True))#, ROOT.RooFit.Range('yRange'), ROOT.RooFit.Range('xRange') )
```
Comment (dntaylor) [inline Limits/python/Models.py:88]: Why this change?

File: HaaLimits/python/HaaLimits2DNew.py [hunk 2]
```diff
@@ -312,7 +312,10 @@ def fitSignals(self,h,region='PP',shift='',**kwargs):
             results[h] = {}
             errors[h] = {}
             integrals[h] = {}
-        if self.YRANGE[1] > 100: initialValuesDCB = self.GetInitialValuesDCB(isKinFit=isKinFit)
+        if self.YRANGE[1] > 100: 
+            if yFitFunc == "DCB_Fix": initialMeans = self.GetInitialDCBMean()
```
Comment (dntaylor) [inline HaaLimits/python/HaaLimits2DNew.py:316]: So, what is the new way to call?

yFitFuncFP = "DCB" or "DCB_Fix"
yFitFuncPP = "DG"

**Comment by dntaylor** [comment]
@kmtos Can you answer my two comments and I'll rerun my limits?

File: Limits/python/Models.py (hunk 1 above)
Comment (kmtos) [reply Limits/python/Models.py:88]: I don't remember even making this change. I can easily revert it.

File: HaaLimits/python/HaaLimits2DNew.py (hunk 2 above)
Comment (kmtos) [reply HaaLimits/python/HaaLimits2DNew.py:316]: DCB = typical DCB fitting
DCB_Fix = the same except additionally fixing the mean. It was one of the test I did to try to improve the DCB. Don't use it, since it didn't have much effect. I just left it in there in case we wanted to do more testing with it.  

You should use:
yFitFuncFP = "DCB"
yFitFuncPP = "DG"

File: Limits/python/Models.py (hunk 1 above)
Comment (kmtos) [reply Limits/python/Models.py:88]: I made the changes and committed them, so this open pull request should have them now.
